ingredient/ingredient-function-app/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@azbake/ingredient-function-app",
+  "description": "Ingredient for deploying Function Apps",
+  "version": "0.0.6",
+  "main": "dist/index.js",
+  "typings": "src/index.ts",
+  "author": "HCHB",
+  "license": "MIT",
+  "scripts": {
+    "upload": "tsc -p . && npm --no-git-tag-version version patch && npm publish --access public",
+    "compile": "tsc -p .",
+    "watch": "tsc -w -p ."
+  },
+  "directories": {
+    "lib": "dist"
+  },
+  "peerDependencies": {
+    "@azbake/core": "0.*",
+    "@azure/ms-rest-nodeauth": "0.9.2"
+  },
+  "devDependencies": {
+    "@azbake/core": "^0.1.44",
+    "@types/node": "^10.12.18",
+    "@azure/ms-rest-nodeauth": "0.9.2"
+  },
+  "dependencies": {
+    "@azbake/arm-helper": "^0.1.44"
+  }
```

Comment:
Crap, sorry I didn't see this before, and it's a bug in our ingredient template for not including this...

Can you add a section to your package.json like:

"publishConfig": {
    "access": "public"
  }

Otherwise your ingredient won't publish during the pipeline if I accept this to master.

Sorry :(. This should be last thing needed!

Reply:
added